metaci/release/utils.py

```diff
@@ -13,12 +13,17 @@
 logger = logging.getLogger(__name__)
 
 
-def implementation_payload(role, config_item):
-    if role and config_item:
+def implementation_payload(role, config_item, release):
+    if role and config_item and release:
         return {
             "description": role,
-            "start_time": role,
-            "end_time": role,
+            "owner": f"{settings.GUS_BUS_OWNER_ID}",
```

Comment:
There's no reason to use an f-string if all it does is include a variable that is already a string:

```suggestion
            "owner": settings.GUS_BUS_OWNER_ID,
```